mantidproject/mantid
# Disable failing Jupyter test

**Description of work.**

This PR deletes a Jupyter test file so that it is not run. 

This is a temporary fix to get our CI/CD back up and running.

I have created an issue to track this and ensure this fix is temporary: #35403

**To test:**

Ensure that `qt/python/mantidqt/mantidqt/widgets/test/test_jupyterconsole.py` is not run, and that all other tests pass.






---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?
- Are the release notes saved in a separate file, using Issue or PR number for file name and in the correct location?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
